dashboard/app/models/blockly.rb

```diff
@@ -135,4 +135,9 @@ def update_ideal_level_source
     return if !self.respond_to?(:solution_blocks) || solution_blocks.blank?
     self.ideal_level_source_id = LevelSource.find_identical_or_create(self, solution_blocks).id
   end
+
+  def embed_blocks(blocks)
```

Comment:
Could we use a more descriptive method and parameter name here, or add a descriptive comment? Just by looking at this method on its own, i can't figure out what Blockly#embed_blocks(blocks) does or what it type of object it expects as a parameter here.